rocketpy/mathutils/function.py

```diff
@@ -110,27 +119,25 @@ def __init__(
         (II) Fields in CSV files may be enclosed in double quotes. If fields
         are not quoted, double quotes should not appear inside them.
         """
-
-        inputs, outputs = Function._validate_inputs_outputs(inputs, outputs)
-
         # initialize variables to avoid errors when being called by other methods
-        self.get_value_opt = None
         self.__polynomial_coefficients__ = None
         self.__akima_coefficients__ = None
         self.__spline_coefficients__ = None
```

Comment:
Do you think we still need these initializations here?
What if we simply remove them?

Reply:
Seems that removing them does not affect the code. Removed them in https://github.com/RocketPy-Team/RocketPy/pull/582/commits/7cd270b956fa020305a6a6bdc45d52cab7283ce7